src/EnergyPlus/OutputProcessor.cc

```diff
@@ -2220,29 +1707,31 @@ namespace OutputProcessor {
         // reporting on each time step.
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int Loop; // Loop Control
         bool PrintTimeStamp;
         int CurDayType;
         auto &op = state.dataOutputProcessor;
+        auto &rf = state.dataResultsFramework->resultsFramework;
+        auto &rfMetersTS = rf->Meters[(int)ReportFreq::TimeStep];
 
-        if (!state.dataResultsFramework->resultsFramework->TSMeters.rDataFrameEnabled()) {
-            state.dataResultsFramework->resultsFramework->initializeMeters(op->EnergyMeters, ReportingFrequency::TimeStep);
+        if (!rfMetersTS.dataFrameEnabled()) {
```

Comment:
OK, so there was a variable called:

```
.rDataFrameEnabled()
```

that appears to be changed to:

```
.dataFrameEnabled()
```

But here the line changed from:

```
rVariablesScanned()
```

to:

```
dataFrameEnabled()
```

The Scanned to Enabled seems suspect, but I think it's just because I would need to dig in a little deeper.  I'll just carry on but thought it was worth noting.

Reply:
Are you sure? It looks like .rDataFrameEnabled in develop.